chall01/osalmine.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python3
+
+import sys
+
+morse = { 'A':'.-', 'B':'-...',
+                    'C':'-.-.', 'D':'-..', 'E':'.',
+                    'F':'..-.', 'G':'--.', 'H':'....',
+                    'I':'..', 'J':'.---', 'K':'-.-',
+                    'L':'.-..', 'M':'--', 'N':'-.',
+                    'O':'---', 'P':'.--.', 'Q':'--.-',
+                    'R':'.-.', 'S':'...', 'T':'-',
+                    'U':'..-', 'V':'...-', 'W':'.--',
+                    'X':'-..-', 'Y':'-.--', 'Z':'--..'}
+
+if len(sys.argv) != 2:
+	print("usage: ./osalmine.py <a-zA-Z string>")
+	quit()
+
+string = sys.argv[1].upper();
```

Comment:
changing string to upper - smart! 
also, I see C-legacy here ';' ;) 
but yeah, I can relate, it's not a deal